ui/imports/shared/views/chat/MessageView.qml

```diff
@@ -445,6 +455,87 @@ Loader {
         }
     }
 
+    Component {
+        id: deletedMessageComponent
+
+        ColumnLayout {
+
+            RowLayout {
+                id: deletedMessage
+                height: 40
```

Comment:
I think we shouldn't set `height` here, it should be calculated automatically based on content size and given margins.